ini.go

```diff
@@ -111,8 +111,15 @@ type LoadOptions struct {
 	KeyValueDelimiters string
 	// PreserveSurroundedQuote indicates whether to preserve surrounded quote (single and double quotes).
 	PreserveSurroundedQuote bool
+	// Debug indicates whether to collect debug information (currently only useful to debug parsing Python-style multiline values).
+	Debug DebugFunc
+	// ReaderBufferSize is the buffer size of the reader in bytes.
+	ReaderBufferSize int
 }
 
+// DebugFunc is the type of function called to log parse events
+type DebugFunc func(args ...interface{})
```

Comment:
How does a user know which arg corresponding to which?

Reply:
I still think `DebugWriter io.Writer` is the cleanest solution here.

Reply:
args should be passed to something like fmt.Println(). A typical implementation would look like this:
```
func Debug(args ...interface{}) {
   fmt.Println(args...)
}
```
This allows to easily plug in any logger or writer.